src/gleam_erlang_ffi.erl

```diff
@@ -109,3 +110,24 @@ delete_directory(Dir) ->
 
 recursive_delete(Dir) ->
     posix_result(file:del_dir_r(Dir)).
+
+get_all_env() ->
+    BinVars = lists:map(fun(VarString) ->
+                                [VarName, VarVal] = string:split(VarString, "="),
+                                {list_to_binary(VarName), list_to_binary(VarVal)}
+                        end, os:getenv()),
+    maps:from_list(BinVars).
```

Comment:
```suggestion
    BinVars = lists:map(fun(VarString) ->
        [VarName, VarVal] = string:split(VarString, "="),
        {list_to_binary(VarName), list_to_binary(VarVal)}
    end, os:getenv()),
    maps:from_list(BinVars).
```

Reply:
Yeah, sorry about this. Erlang ships with some ELisp the provide formatting, syntax highlighting, etc for Emacs. This is how that ELisp wanted this formatted :laughing: I meant to leave a comment about how off-the-wall crazy it is.